vector/src/main/java/im/vector/app/features/home/room/detail/timeline/helper/VoiceMessagePlaybackTracker.kt

```diff
@@ -26,8 +26,17 @@ class VoiceMessagePlaybackTracker @Inject constructor() {
 
     private val mainHandler = Handler(Looper.getMainLooper())
     private val listeners = mutableMapOf<String, Listener>()
+    private val activityListeners = mutableMapOf<String, ActivityListener>()
     private val states = mutableMapOf<String, Listener.State>()
 
+    fun trackActivity(key: String, listener: ActivityListener) {
```

Comment:
Maybe rename `key` to `id` for symmetry with `fun unTrackActivity()` and the other methods related to listeners